Makefile

```
@@ -7,9 +7,17 @@ IMAGE_REGISTRY 	?= ghcr.io
 IMAGE_REPO     	?= kedacore
 VERSION 		?= main
 
-IMAGE_OPERATOR 		?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-operator:$(VERSION)
-IMAGE_INTERCEPTOR	?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-interceptor:$(VERSION)
-IMAGE_SCALER		?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-scaler:$(VERSION)
+IMAGE_OPERATOR_BASE 	?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-operator
+IMAGE_INTERCEPTOR_BASE	?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-interceptor
+IMAGE_SCALER_BASE		?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-scaler
+
+IMAGE_OPERATOR_VERSION		?= ${IMAGE_OPERATOR_BASE}:$(VERSION)
+IMAGE_INTERCEPTOR_VERSION	?= ${IMAGE_INTERCEPTOR_BASE}:$(VERSION)
+IMAGE_SCALER_VERSION		?= ${IMAGE_SCALER_BASE}:$(VERSION)
+
+IMAGE_OPERATOR_SHA		?= ${IMAGE_OPERATOR_BASE}:$(GIT_COMMIT_SHORT)
+IMAGE_INTERCEPTOR_SHA	?= ${IMAGE_INTERCEPTOR_BASE}:$(GIT_COMMIT_SHORT)
+IMAGE_SCALER_SHA		?= ${IMAGE_SCALER_BASE}:$(GIT_COMMIT_SHORT)
```

Let's correctly use "image" and "tag" concepts
```suggestion
IMAGE_OPERATOR 	?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-operator
IMAGE_INTERCEPTOR	?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-interceptor
IMAGE_SCALER		?= ${IMAGE_REGISTRY}/${IMAGE_REPO}/http-add-on-scaler

IMAGE_OPERATOR_VERSIONED_TAG		?= ${IMAGE_OPERATOR}:$(VERSION)
IMAGE_INTERCEPTOR_VERSIONED_TAG	?= ${IMAGE_INTERCEPTOR}:$(VERSION)
IMAGE_SCALER_VERSIONED_TAG		?= ${IMAGE_SCALER}:$(VERSION)

IMAGE_OPERATOR_SHA_TAG		?= ${IMAGE_OPERATOR}:$(GIT_COMMIT_SHORT)
IMAGE_INTERCEPTOR_SHA_TAG	?= ${IMAGE_INTERCEPTOR}:$(GIT_COMMIT_SHORT)
IMAGE_SCALER_SHA_TAG		?= ${IMAGE_SCALER}:$(GIT_COMMIT_SHORT)
```

---

sure